### ISSUE TITLE
[Problem] App Icon Selection Is Broken

### ISSUE THREAD

**Issue by theanthonydavenport** [ghl3m0n/FuckElon#13]
![IMG_2266](https://github.com/user-attachments/assets/a5daf77a-5a47-4c91-b981-64e17b685bb9)

**Comment by ikastolerohk** [comment]
Yeah, never worked for me. I can't even select another icon, the checkbox just disappears. 
Iphone SE 2022 w/  iOS 16.4.1 in Trollstore btw. Everything else works as a treat!
![d1495b86-bf3b-418c-a192-b2feef1c480d](https://github.com/user-attachments/assets/068e66cc-d915-4c66-81cd-9c388849b673)

**Comment by theanthonydavenport** [comment]
I use an iPhone 15 Pro Max on iOS 18.1 beta.
